Create an issue template for gathering details of bugs (#1)
omise/.github: ISSUE_TEMPLATE/bug-report.md

---
name: Bug Report
about: Report a bug
title: ''
labels: ''
assignees: '@omise/maintainers'

---

## Expected behavior

_Describe what you expected to happen._

## Actual behavior

_Describe what actually happened._

## Steps to reproduce the issue

_List the steps to reproduce the issue._

> 

Reviewer: What's the purpose of this blank line?

Author: I removed it.

> ## Logs/Error messages

Reviewer: ```suggestion
## Logs/error messages
```

> _Attach or paste appropriate logs and error messages to help us debug better._

Reviewer: ```suggestion
_Attach or paste appropriate logs and error messages to help us debug more easily._
```

> ## Name and Version Information

Reviewer: ```suggestion
## Name and version information
```

_Specify the name and version of the following components:_

> _* Development Environment (For example, XCode version 12.0)._

Reviewer: ```suggestion
_* Development Environment (for example, XCode version 12.0)._
```

> _* SDK (For example, Omise-iOS version 4.25)._

Reviewer: ```suggestion
_* SDK (for example, Omise-iOS version 4.25)._
```

> _* Package Manager (For example, SwiftPM version 0.6.0)._

Reviewer: ```suggestion
_* Package Manager (for example, SwiftPM version 0.6.0)._
```

> _* Operating System (For example, iOS version 17.1.1)._

Reviewer: ```suggestion
_* Operating System (for example, iOS version 17.1.1)._
```

> _* Browser, if it is a web issue (For example, Firefox version 119.0.1)._

Reviewer: ```suggestion
_* Browser, if it is a web issue (for example, Firefox version 119.0.1)._
```
